### PR TITLE
Helm chart and instructions for addon backdoor deployment

### PR DESCRIPTION
**Pull request by rashmichandrashekar**
Added helm chart similar to the one in AKS-RP with the templates and values model to keep it consistent with AKS-RP.
Instructions on how to use the chart.

### CODE REVIEW

**Review by Sohamdg081992 — APPROVED** [review]

File: otelcollector/deploy/addon-chart/Readme.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+### **Step 1: Go to addon-chart directory**
```
Comment (gracewehner) [inline otelcollector/deploy/addon-chart/Readme.md:1]: Maybe we could also doc as a pre-req to follow the ARM template deployment instructions (but set azuremonitormetrics to false) to deploy the other artifacts first?

**Review by gracewehner — APPROVED** [review]

**Review by Sohamdg081992 — DISMISSED** [review]

File: otelcollector/deploy/addon-chart/Readme.md (hunk 1 above)
Comment (rashmichandrashekar) [reply otelcollector/deploy/addon-chart/Readme.md:1]: Yes. I have a couple more instructions to add. Let me add  it and you can reapprove. Thanks!

Comment (rashmichandrashekar) [reply otelcollector/deploy/addon-chart/Readme.md]: Updated the readme. pls take a look.

**Review by gracewehner — APPROVED** [review]
Thanks Rashmi!

**Pull request merged**